src/index.ts

```diff
@@ -55,6 +55,8 @@ const server = app.listen(process.env.PORT ?? 3000, () => {
 });
 onExit(() => new Promise<void>(resolve => server.close(() => resolve())));
 
-await createBiddingRules(1, await bot.accountset.client, process.env.BIDDING_RULES_PATH!);
+
+const rulesContent = readJsonFileOrNull(process.env.BIDDING_RULES_PATH!) || {};
```

Comment:
This code shouldn't be in index. It was test code I removed that somehow crept back in

Reply:
Removed